src/orga/governance.rst

```diff
@@ -102,6 +106,68 @@ To call for a standard vote, here is a template PR comment:
     To vote please leave Approve (yea) or Request Changes (nay) reviews.
     If you would like changes to the current language please leave a comment or push to this branch.
     This vote will end on {date}.
+    
+----
+:Posting results: To maintain the historical record, the outcome of any vote should be 
```

Comment:
I read the difference between standard and sensitive as just whether the voting is anonymous or not.  I think vote totals in the historical record are all that is necessary - not names.  I see standard and sensitive votes as equivalent for this purpose.  Am I missing something here?

Reply:
In other words, *all* votes are public, it's just that some of them are voted on anonymously, while others take place in a way where names are shown.

Reply:
Hmm, this is interesting actually. The reason we set up sensitive votes as I remember it was so that a new core member wouldn't come in knowing "oh @dougalsutherland voted against me, weird"; it's marginally better to mask the name and know "oh two people voted against me", but still weird?

Reply:
We can discuss whether/how much of the "sensitive" vote results can or should be made public. But we should do that in a separate discussion/PR.

My intention is just to make sure we don't touch too many different topics here.
The description of "sensitive votes" has
> Sensitive voting items are ones where the results of the
vote should remain private to the voters after the vote has occurred.

which does not clarify whether the individual or the overall vote results are meant to "remain private to the voters". If we (in this first step, i.e., this PR) restrict the result posting to non-sensitive votes for now, we don't contradict any interpretation of that sentence.

(The anonymity part
> Sensitive votes should take place on a secure anonymous voting platform
in order to retain election integrity and anonymity.

is a little bit clearer due to the "retain election integrity and anonymity" phrase.)

Reply:
Fair enough.  Happy to merge and continue that later.